### PR TITLE
Add base image

### PR DESCRIPTION
**Pull request by LKHN**
Signed-off-by: Elkhan Mammadli <elkhan.mammadli@protonmail.com>

### CODE REVIEW

File: kickstarts/almalinux-8-base.x86_64.ks [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# https://fedoraproject.org/wiki/Changes/Glibc_locale_subpackaging for details
+echo '%_install_langs en_US.UTF-8' > /etc/rpm/macros.image-language-conf
+
+# force each container to have a unique machine-id
+> /etc/machine-id
+
+# create tmp directories because there is no tmpfs support in Docker
+umount /run
+systemd-tmpfiles --create --boot
+
```
Comment (srbala) [inline kickstarts/almalinux-8-base.x86_64.ks:80]: I believe following lines may needs to be added.

```sh
# disable login prompt and mounts
systemctl mask console-getty.service \
               dev-hugepages.mount \
               getty.target \
               systemd-logind.service \
               sys-fs-fuse-connections.mount \
               systemd-remount-fs.service
```

Comment (LKHN) [reply kickstarts/almalinux-8-base.x86_64.ks:80]: if systemd not starting when container started, how this lines will work? 🤔

Comment (srbala) [reply kickstarts/almalinux-8-base.x86_64.ks:80]: These are definitively required. It creates link to `/dev/null` entries in `/etc/systemd/system` folder.

`redhat/ubi8` 
```sh
lrwxrwxrwx 1 root root    9 Aug  3 16:25 console-getty.service -> /dev/null
lrwxrwxrwx 1 root root   41 Aug  3 16:25 default.target -> /usr/lib/systemd/system/multi-user.target
lrwxrwxrwx 1 root root    9 Aug  3 16:25 dev-hugepages.mount -> /dev/null
lrwxrwxrwx 1 root root    9 Aug  3 16:25 getty.target -> /dev/null
drwxr-xr-x 2 root root 4096 Aug  3 16:25 getty.target.wants
drwxr-xr-x 2 root root 4096 Aug  3 16:25 multi-user.target.wants
lrwxrwxrwx 1 root root    9 Aug  3 16:25 sys-fs-fuse-connections.mount -> /dev/null
lrwxrwxrwx 1 root root   39 Aug  3 16:25 syslog.service -> /usr/lib/systemd/system/rsyslog.service
lrwxrwxrwx 1 root root    9 Aug  3 16:25 systemd-logind.service -> /dev/null
lrwxrwxrwx 1 root root    9 Aug  3 16:25 systemd-remount-fs.service -> /dev/null
drwxr-xr-x 2 root root 4096 Aug  3 16:25 timers.target.wants
```

on the new system
```sh
lrwxrwxrwx 1 root root   41 Sep  9 21:36 default.target -> /usr/lib/systemd/system/multi-user.target
drwxr-xr-x 2 root root 4096 Sep  9 21:36 getty.target.wants
drwxr-xr-x 2 root root 4096 Sep  9 21:36 multi-user.target.wants
lrwxrwxrwx 1 root root   39 Sep  9 21:36 syslog.service -> /usr/lib/systemd/system/rsyslog.service
drwxr-xr-x 2 root root 4096 Sep  9 21:36 timers.target.wants
```

File: kickstarts/almalinux-8-base.aarch64.ks [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+# force each container to have a unique machine-id
+> /etc/machine-id
+
+# create tmp directories because there is no tmpfs support in Docker
+umount /run
+systemd-tmpfiles --create --boot
+
+# disable login prompt, mounts and fix: https://bugzilla.redhat.com/show_bug.cgi?id=1472439
+systemctl mask systemd-remount-fs.service \
```
Comment (srbala) [inline kickstarts/almalinux-8-base.aarch64.ks:82]: @LKHN Changes looks similar to `ubi-init`, has 4 `extra` entries in both ks fiels any reason? 🤔 
```
systemd-remount-fs.service 
systemd-udev-trigger.service 
systemd-udevd.service 
systemd-random-seed.service 
```

Comment (srbala) [reply kickstarts/almalinux-8-base.aarch64.ks:82]: Simple use 
```
systemctl mask console-getty.service \
               dev-hugepages.mount \
               getty.target \
               systemd-logind.service \
               sys-fs-fuse-connections.mount \
               systemd-remount-fs.service
```

**Review by srbala — APPROVED** [review]
LGTM

**Pull request merged**